installation/index.md

```diff
@@ -36,13 +36,16 @@ The previously mentioned [openHABian](openhabian.html) can also be used to kicks
 
 ## Prerequisites
 
-Make sure that you have an up to date version of the [Oracle Java Runtime Environment](https://java.com/en/download/manual.jsp) on your host system installed.
-A famous choice is OpenJDK, however, Oracle Java Version 8 is recommended for openHAB.
-openHAB may be unstable and slower executed by OpenJDK.
+Make sure that you have an up to date Java Runtime Environment (JRE) installed on your host system. The [Oracle JRE](https://java.com/en/download/manual.jsp)
```

Comment:
I agree with @kaikreuzer, let's make Zulu the recommended version.
For now I would however also point to Oracle Java as an equal option, if already installed for example.
@stephenbonar maybe you can come up with a nice statement

Also (others will laugh) please make sure your new paragraphs are one line per sentence, just like the existing text. Thanks!